themes/_core/js/cart.js

```diff
@@ -29,34 +29,37 @@ $(document).ready(() => {
   prestashop.on('updateCart', (event) => {
     prestashop.cart = event.reason.cart;
     var getCartViewUrl = $('.js-cart').data('refresh-url');
-    var requestData = {};
+    
+    if (getCartViewUrl) {
```

Comment:
You can avoid adding a indentation to the whole method by adding instead 

```js
if (!getCartViewUrl) {
  return;
}
```

The diff would be also easier to read. :)

Reply:
Done